data/campaigns/emp03.wmf/scripting/texts.lua

```diff
@@ -372,10 +372,10 @@ amalea_4 = {
       _([[So, let’s think this over. To weave cloth we need some wool from a sheep farm. The sheepfarm itself needs wheat and water to work. So, let’s find a space for a big building and build a farm first.]])
       .. paragraphdivider() ..
       -- TRANSLATORS: Amalea
-      _([[After the production of enough wheat, we can dismantle the farm and build a sheep farm to produce wool. Later on, we could build a weaving mill in the same spot to use our wool for weaving cloth. But for the moment being, let’s start by producing a reasonable amount of wheat.]])
+      _([[After the production of enough wheat, we can dismantle the farm and build a sheep farm to produce wool. When the sheep farm starts working, we could find or free up space for a medium building to build a weaving mill to use our wool for weaving cloth. But for the moment being, let’s start by producing a reasonable amount of wheat.]])
       .. paragraphdivider() ..
       -- TRANSLATORS: Amalea
-      _([[I think a stock of 50 sheaves of wheat would be enough for starters. Perhaps we will have to adjust the target quantity for wheat to achieve this.]]))
+      _([[I think a stock of 50 sheaves of wheat would be enough for starters. We do not have a target quantity for wheat, because the farmer does not need other wares to produce it. All we have to do is watch our stock of wheat grow.]]))
```

Comment:
well, this one was in fact meant as a hoax to the player. However I admit that this is not very kind from me and could lead to some complaints in the forums.   
So I am happy with the chage but I would be happy as well with deleting the original second sentence conpletely.

Reply:
I thought about removing it, but as this is still kind of a teaching scenario, I thought some explanation wouldn't hurt.

Reply:
ok, I'll merge it then